pkgs/development/interpreters/spidermonkey/91.nix

```diff
@@ -39,12 +41,11 @@ stdenv.mkDerivation rec {
     perl
     pkg-config
     python3
-    rust-cbindgen
```

Comment:
Is this no longer required?

Reply:
It's not supposed to be necessary according to the [configure](https://github.com/mozilla/gecko-dev/blob/cc03ccd836330833a8d20fa764f780bd5835477e/build/moz.configure/bindgen.configure#L9) and seems to be marked broken in darwin.